tests/ProfileAsync.Tests.ps1

```diff
@@ -2,6 +2,7 @@ BeforeAll {
     $TestRoot = $PSScriptRoot
     $ModuleBase = $PSScriptRoot | Split-Path
     $PesterBase = (Get-Module Pester).ModuleBase
+    $ContainerRunner = 'docker', 'podman' | ForEach-Object { Get-Command $_ -ErrorAction SilentlyContinue } | Select-Object -Expand Name
```

Comment:
Fails for me, with both installed, with `CommandNotFoundException: The term 'docker podman' is not recognized...`

Could you add `-First 1` to the Select-Object, to fix that? I think docker should take precedence when both are installed.